### ISSUE TITLE
submodule error on fresh clone

### ISSUE THREAD

**Issue by loriab** [psi4/psi4public#189]
All was well on psi4/psi4 circa 7913b38 early morning 15 Nov, but after pub/priv sync, by early morning 16 Nov, fresh recursive clones (```git clone --recursive https://github.com/psi4/psi4.git```) are giving errors
like below. @robertodr, could you check that the current submodule reference is what it ought it be? Thanks.

```
Cloning into 'psi4'...
...
Resolving deltas: 100% (80379/80379), done.
Submodule 'interfaces/pcmsolver' (https://github.com/PCMSolver/pcmsolver.git) registered for path 'interfaces/pcmsolver'
Cloning into 'interfaces/pcmsolver'...
...
Resolving deltas: 100% (13625/13625), done.
fatal: reference is not a tree: 9bc8e049f14260a4c1b4b2bda898a16850907b1c
Unable to checkout '9bc8e049f14260a4c1b4b2bda898a16850907b1c' in submodule path 'interfaces/pcmsolver'
```

**Comment by robertodr** [comment]
I have updated the reference on my branch and will file a pull request (on private) as soon a I manage to build and run some tests. For the moment I have updated the wiki (`git clone ---> `git clone --recursive`)

**Comment by loriab** [comment]
Thanks. Good idea updating the wiki.

**Comment by robertodr** [comment]
PR just filed.

**Comment by loriab** [comment]
Great, PR accepted, tested, synched btwn pub/priv, and now the binary builder can get to work. closed by 6c5512288ac3d16f2e3dee817b30cec4b9cbf9a5
